pkg/deploy/generator/resources.go

```diff
@@ -556,6 +558,66 @@ func (g *generator) lb() *arm.Resource {
 	}
 }
 
+func (g *generator) lbAlerts() []*arm.Resource {
+	return []*arm.Resource{
+		{
+			Resource: mgmtmonitor.ActionGroupResource{
+				ActionGroup: &mgmtmonitor.ActionGroup{
+					Enabled:        to.BoolPtr(true),
+					GroupShortName: to.StringPtr("lbhealth"),
+				},
+				Name:     to.StringPtr("rp-lb-health-ag"),
```

Comment:
suggest finding generic names here in case we use it for multiple items later

Reply:
Changed to: 					
```GroupShortName: to.StringPtr("rphealth"),``` and ```Name:     to.StringPtr("rphealth-ag"),```